doc/cupertino-controls-styles.md

```diff
@@ -0,0 +1,21 @@
+# Cupertino Controls Styles
+Control|Style Key
+-|-
+`Button`|CupertinoButtonStyle<br>CupertinoContainedButtonStyle
+`CalendarView`|CupertinoCalendarViewStyle
+`CalendarDatePicker`|CupertinoCalendarDatePickerStyle
```

Comment:
in vscode select the entire table below `-|-` and f1 -> sort line ascending:
```suggestion
`CalendarDatePicker`|CupertinoCalendarDatePickerStyle
`CalendarView`|CupertinoCalendarViewStyle
```

Reply:
Thanks for the tip @Xiaoy312 will do it ;) !